flottform/forms/src/flottform-channel-client.ts

```diff
@@ -198,12 +263,21 @@ export class FlottformChannelClient extends EventEmitter<Listeners> {
 			}
 		};
 	};
+	/**
+	 * Stops polling for ICE candidates.
+	 */
 	private stopPollingForIceCandidates = async () => {
```

Comment:
I think this is obvious. Since this is private, I don't think we need a comment for those